### DISCUSSION TITLE
Finch not working in /tmp folder

### DISCUSSION THREAD

**Discussion by eshanjain** [runfinch/finch#989] [Q&A] [answered]
I have a 'finch' folder inside the /tmp folder containing a 'dockerfile'. When I run the command **finch build . -t my-image**, I get the following error:
```
bash: line 1: cd: /tmp/finch: No such file or directory
FATA[0000] lstat /Users/myuser/Containerfile: no such file or directory 
FATA[0000] exit status 1
```

When I copy the above 'finch' folder to the 'myuser' directory, the command **finch build . -t my-image** runs successfully.

Any idea how to run the **finch build . -t my-image** command inside the /tmp folder?

**Comment by estesp** [comment, marked as answer]
only the Mac user's home directory is auto-mounted inside the Finch VM; but you can add more directories as needed through the Finch configuration; the snippet from the docs on adding additional directories (from [here](https://github.com/runfinch/finch?tab=readme-ov-file#configuration)):
```
# additional_directories: the work directories that are not supported by default. In macOS, only home directory is supported by default. 
# For example, if you want to mount a directory into a container, and that directory is not under your home directory, 
# then you'll need to specify this field to add that directory or any ascendant of it as a work directory. (optional)
additional_directories:
  # the path of each additional directory.
  - path: /Volumes
```
